#!/usr/bin/env python

PACKAGE = "concert_conductor"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("auto_invite", bool_t, 0, "automatically invite clients", False)
gen.add("local_clients_only", bool_t, 0, "invite clients from other pc's on the network, used for simulations.", False)
gen.add("oblivion_timeout", int_t, 0, "time before a bad, gone client is removed from the index.", 3600, 0)
gen.add("service_timeout", double_t, 0, "waiting timeout for status, start_rapp, and stop_rapp.", 5.0, 0, 100.0)

exit(gen.generate(PACKAGE, "concert_conductor_params", "params"))
